### PR TITLE
Update submodules 'ot-br-posix' and 'openthread' for OTBR PD REF support

### PR DESCRIPTION
**Pull request by yangsong-cnyn**
* ot-br-posix 7f04c183b04dc490a40bf34f72be7a1b79fa1fbf
* openthread deb35b0c257f6f530cf55c89546772ab867e8041
* ot-nrf528xx f485040d87aa42e834bb24b5bf8115679d972ccd

turn on prefix delegation for 1.4

### CODE REVIEW

File: script/otbr-setup.bash [hunk 1]
```diff
@@ -166,6 +167,7 @@ elif [ "${REFERENCE_RELEASE_TYPE?}" = "1.4" ]; then
                 'BORDER_ROUTING=1'
                 'NAT64=1'
                 'DNS64=1'
+                'DHCPV6_PD_REF=1'
```
Comment (superwhd) [inline script/otbr-setup.bash:170]: Just curious: @sunytt, Do you have any idea why we need to repeat OTBR features' config as part of `LOCAL_OPTIONS`?

**Review by superwhd — APPROVED** [review]
LGTM 👍

**Comment by sunytt** [comment]
Could you log this update in the CHANGLOG file? Thanks @yangsong-cnyn

**Comment by sunytt** [comment]
When ot and otbr submodules are updated, may also update the ot-nrf528xx to contain latest changes.

**Comment by yangsong-cnyn** [comment]
> Could you log this update in the CHANGLOG file? Thanks @yangsong-cnyn

updated

File: script/otbr-setup.bash (hunk 1 above)
Comment (sunytt) [reply script/otbr-setup.bash:170]: I don't know the reason why these local options are repeated from the beginning, I think different platforms can share the same definition. Could be an improvement for future.

**Review by sunytt — APPROVED** [review]
LGTM, thanks

**Review by jwhui — APPROVED** [review]

**Pull request merged**